tools/sync_test_server/ros-testing-server.js

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env nodejs
+
+var winston = require('winston');//logging
+const temp = require('temp');
+const spawn = require('child_process').spawn;
+var http = require('http');
+var dispatcher = require('httpdispatcher');
+
+// Automatically track and cleanup files at exit
+temp.track();
+
+if (process. argv. length <= 2) {
+    console.log("Usage: " + __filename + " somefile.log");
+    process.exit(-1);
+}
+const logFile = process.argv[2];
+winston.level = 'debug';
+winston.add(winston.transports.File, { filename: logFile });
+
+const PORT = 8888;
+
+function handleRequest(request, response) {
+    try {
+        //log the request on console
+        winston.log(request.url);
+        //Disptach
+        dispatcher.dispatch(request, response);
+    } catch(err) {
+        console.log(err);
+    }
+}
+
+var syncServerChildProcess;
+
+function startRealmObjectServer() {
+    temp.mkdir('ros', function(err, path) {
+        if (!err) {
+            winston.info("Starting sync server in ", path);
+            syncServerChildProcess = spawn('realm-object-server',
+                    ['--root', path,
+                    '--configuration', '/configuration.yml']);
+            // local config:
+            syncServerChildProcess.stdout.on('data', (data) => {
+                winston.info(`stdout: ${data}`);
+            });
+
+            syncServerChildProcess.stderr.on('data', (data) => {
+                winston.info(`stderr: ${data}`);
+            });
+
+            syncServerChildProcess.on('close', (code) => {
+                winston.info(`child process exited with code ${code}`);
+            });
+        }
+    });
+}
+
+// start sync server
+dispatcher.onGet("/start", function(req, res) {
+    startRealmObjectServer();
```

Comment:
should we try to close a previously started server that didn't close properly (ex `/stop`) not called.
```javascript
if (syncServerChildProcess != null) {
 syncServerChildProcess.kill();
}
```